pkg/sgpp/utils.go

```diff
@@ -30,3 +38,68 @@ func LoadEnforcingModeFromEnv() EnforcingMode {
 		return DefaultEnforcingMode
 	}
 }
+
+func AddFeatureToCNINode(ctx context.Context, k8sClient client.Client, nodeName string, featureName rc.FeatureName, featureValue string) error {
```

Comment:
Since this package is for `sgpp`, and this feature is going to be more general, I am wondering if this belongs in a different package. Maybe `k8sapi` or `utils`?

Reply:
According to Ellis' suggestion, I think it makes sense to have this function associated to IPAM.